packages/slate-hyperscript/test/index.test.js

```diff
@@ -2,7 +2,7 @@
  * Dependencies.
```

Comment:
Can we change the config to not have to add the `.test.js` suffix?

Reply:
I have not yet found out a way of that unfortunately.  We have to have either `__test__` or `.test.js`

Reply:
@zhujinxuan I think it can happen with the `testMatch` configuration option? https://jestjs.io/docs/en/configuration#testmatch-array-string

Reply:
Great!  That is what I am looking for.  I will fix it minutes later.

Reply:
Fixed~  Thank you